src/scripts/main.js

```diff
@@ -1,29 +1,58 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
```

Comment:
Consider adding a function description. It's a good practice to describe what the function does, its parameters and what it returns. This makes the code more understandable and maintainable.